testlib/src/test/java/com/diffplug/spotless/scala/ScalaFmtStepTest.java

```diff
@@ -113,18 +92,10 @@ void invalidConfiguration() throws Exception {
 		File invalidConfFile = createTestFile("scala/scalafmt/scalafmt.invalid.conf");
 		Provisioner provisioner = TestProvisioner.mavenCentral();
 
-		InvocationTargetException exception;
-
-		exception = assertThrows(InvocationTargetException.class,
-				() -> StepHarness.forStep(ScalaFmtStep.create("1.1.0", provisioner, invalidConfFile)).test("", ""));
-		assertThat(exception.getTargetException().getMessage()).contains("Invalid fields: invalidScalaFmtConfigField");
-
-		exception = assertThrows(InvocationTargetException.class,
-				() -> StepHarness.forStep(ScalaFmtStep.create("2.0.1", provisioner, invalidConfFile)).test("", ""));
-		assertThat(exception.getTargetException().getMessage()).contains("Invalid field: invalidScalaFmtConfigField");
+		NoSuchElementException exception;
 
-		exception = assertThrows(InvocationTargetException.class,
+		exception = assertThrows(NoSuchElementException.class,
```

Comment:
The exception has changed from`InvocationTargetException` to `NoSuchElementException` due to the fact that we are no longer using reflection and instead we are directly calling the method. It was likely that previously there was a nested `NoSuchElementException` inside of the `InvocationTargetException` (which. due to the `contains()` would have still passed the test).